components/cam/src/physics/cam/phys_control.F90

```diff
@@ -343,6 +360,14 @@ subroutine phys_ctl_readnl(nlfile)
       end if
    end if
 
+   ! Check settings for MMF_microphysics_scheme
+   if ( .not.(MMF_microphysics_scheme .eq. 'm2005'   .or. &
+              MMF_microphysics_scheme .eq. 'sam1mom' .or. &
+              MMF_microphysics_scheme .eq. unset_str )) then
```

Comment:
Should we encapsulate this in a `if (use_MMF)` and then pull out the check against `unset_str` here? If it's still `unset_str`, then I think we failed to set the microphysics scheme, and we should abort right here.

Reply:
sure.

Reply:
this check actually says unset_str is a valid value, which I guess is why we didn't need the "use_MMF". So I think I can just remove the check for unset_str.